lib/utils/test-task-mixin.js

```diff
@@ -1,25 +1,9 @@
 'use strict';
 
-function setTestemFile(options, file = 'testem-electron.js') {
-  if (typeof options === 'object') {
-    options.file = file;
-  }
-
-  return options;
-}
-
 module.exports = {
-  init() {
-    // Make sure to use our version of testem, rather than ember-cli's
-    this.testem = this.testem || new (require('testem'))();
-    this._super(...arguments);
-  },
-
-  defaultOptions() {
-    return setTestemFile(this._super(...arguments));
-  },
-
-  testemOptions() {
-    return setTestemFile(this._super(...arguments)) || this.defaultOptions(...arguments);
+  run(options) {
+    options.configFile = 'testem-electron.js';
```

Comment:
Shouldn't we still allow the consumer to override if they want? e.g. `options.configFile = options.configFile || 'testem-electron.js';`

Reply:
Good call!